build/build.go

```diff
@@ -463,7 +463,16 @@ func toSolveOpt(ctx context.Context, node builder.Node, multiDriver bool, opt Op
 		}
 	}
 	if _, ok := opt.Attests["attest:provenance"]; !ok && supportsAttestations {
-		so.FrontendAttrs["attest:provenance"] = "mode=min,inline-only=true"
+		var noProv bool
+		if v, ok := os.LookupEnv("BUILDX_NO_DEFAULT_PROVENANCE"); ok {
```

Comment:
I recall we have https://github.com/docker/buildx/blob/78058ce5f301fb5f7acb31eaee16fc85e37911dc/build/build.go#L386 to inline cache metadata to image config. I wonder if the var should be named `BUILDKIT_NO_DEFAULT_PROVENANCE` instead for consistency even if `BUILDKIT_INLINE_CACHE` should have been `BUILDX_INLINE_CACHE` imo as this is var is not handled in BuildKit but Buildx.

Reply:
Yeah, we also have [`BUILDX_NO_DEFAULT_LOAD`](https://github.com/docker/buildx/blob/04804ff3551b44d2d930be669aceaf2dd41b1f8b/build/build.go#L1698-L1708) though :thinking: 

Consistency isn't great here, though I'd be tempted to go with the `BUILDX_` prefix if possible.

I wonder if we should make the variable name more generic though, something like `BUILDX_NO_DEFAULT_ATTESTATIONS` - just in case we ever went the route of making some other attestations default as well (currently not planned, but better to have the breathing room IMHO).

Reply:
@cpuguy83 wdyt?

Reply:
`BUILDX_NO_DEFAULT_ATTESTATIONS` lgtm

Reply:
Updated.